tests/ord-service/tests/subscription_flow_test.go

```diff
@@ -1081,3 +1087,41 @@ func fixAppTemplateInputWithDefaultDistinguishLabelAndSubdomainRegion(name strin
 	input.Placeholders = append(input.Placeholders, &graphql.PlaceholderDefinitionInput{Name: "subdomain"}, &graphql.PlaceholderDefinitionInput{Name: "region"})
 	return input
 }
+
+func getExpectedFormationDetailsAssignmentIDForRuntimeProvider(t require.TestingT, ctx context.Context, tenantID, consumerAppID, formationID string) string {
+	listFormationAssignmentsRequest := fixtures.FixListFormationAssignmentRequest(formationID, 200)
+	assignmentsPage := fixtures.ListFormationAssignments(t, ctx, certSecuredGraphQLClient, tenantID, listFormationAssignmentsRequest)
+	assignments := assignmentsPage.Data
+	require.NotEmpty(t, assignments)
+	expectedFormationDetailsAssignmentID := ""
+	for _, assignment := range assignments {
+		if assignment.Source == consumerAppID && assignment.TargetType == graphql.FormationAssignmentTypeRuntimeContext {
```

Comment:
Maybe instead of checking only the runtime context type, we could check for a specific runtime context ID, similar to the application template provider/app ID. That way we also could unify both functions(`` and ``) and use only one with source=`consumerAppID` and target=`providerEntityID` or something similar.
P.S. It's something minor so I'll approve the PR